packages/elements/src/photon-patient-dialog/index.tsx

```diff
@@ -47,7 +47,7 @@ customElement(
       const event = new CustomEvent('photon-patient-closed', {
         composed: true,
         bubbles: true,
-        detail: {},
+        detail: {}
```

Comment:
do we have a linting file in here?

Reply:
we do, it's global to the repo now, was gonna run  it on a separate PR so it doesn't blow up the commits